templates/header/package_navigation.html

```diff
@@ -27,7 +27,7 @@ <h1 id="crate-title">
                         {{ title }}
                     {%- else -%}
                         {{ metadata.name }} {{ metadata.version }}
-                        {{ "copy" | far(id="clipboard", aria_label="Copy crate name and version information", fa=true) }}
+                        <span id="clipboard" class="fa-svg fa-svg-fw" aria-label="Copy crate name and version information">{%- include "clipboard.svg" -%}</span>
```

Comment:
We probably want this label as a title? As mentioned on <https://github.com/rust-lang/rust/issues/84639>/<https://github.com/rust-lang/rust/issues/84986> it's not obvious what this button does. (I personally don't know enough about ARIA to know whether the `aria-label` would still be necessary as well or not).

Reply:
Okay, that's done.